apis/nucleus/src/components/listbox/components/ListBoxRowColumn/__tests__/list-box-row-in-div.test.jsx

```diff
@@ -0,0 +1,201 @@
+import { createTheme, ThemeProvider } from '@nebula.js/ui/theme';
+import { Box } from '@mui/material';
+import ListBoxRowColumn from '..';
+import useTempKeyboard from '../../useTempKeyboard';
+import * as screenReader from '../../ScreenReaders';
+
+const getRow = (i) => screen.getAllByTestId('listbox.item')[i].firstChild;
+
+describe('check keyboard navigation rendering with multiple rows in the in-built headless browser', () => {
+  let style;
+  let data;
+  let theme;
+  let actions;
+  let containerRef;
+
+  beforeEach(() => {
+    jest.spyOn(screenReader, 'getValueLabel').mockReturnValue('ariaLabel');
+    const mockUseState = jest.spyOn(React, 'useState');
+    mockUseState.mockReturnValue([true, jest.fn()]);
+    const keyboard = useTempKeyboard({ containerRef, enabled: false });
+
+    actions = {
+      select: jest.fn(),
+      selectAll: jest.fn(),
+      onCtrlF: jest.fn(),
+      confirm: jest.fn(),
+      cancel: jest.fn(),
+    };
+    theme = createTheme('dark');
+    style = {};
+    data = {
+      onMouseDown: jest.fn(),
+      onMouseUp: jest.fn(),
+      onMouseEnter: jest.fn(),
+      onClick: jest.fn(),
+      keyboard,
+      actions,
+      isSingleSelect: false,
+      checkboxes: false,
+      frequencyMode: 'value',
+      dataOffset: 0,
+      focusListItems: {
+        first: undefined,
+        last: undefined,
+      },
+      layoutOptions: {
+        layoutOrder: 'row',
+      },
+      isModal: jest.fn().mockReturnValue(false),
+      pages: [
+        {
+          qArea: {
+            qLeft: 0,
+            qTop: 0,
+            qWidth: 0,
+            qHeight: 3,
+          },
+          qMatrix: [
+            [
+              {
+                qState: 'A',
+                qFrequency: '1',
+                qText: 'A',
+                qElemNumber: 0,
+              },
+            ],
+            [
+              {
+                qState: 'A',
+                qFrequency: '2',
+                qText: 'B',
+                qElemNumber: 1,
+              },
+            ],
+            [
+              {
+                qState: 'A',
+                qFrequency: '3',
+                qText: 'C',
+                qElemNumber: 2,
+              },
+            ],
+          ],
+        },
+      ],
+    };
+  });
+
+  it('focus should walk down and up the rows when pressing arrows keys (right, down, up, left)', async () => {
+    render(
+      <ThemeProvider theme={theme}>
+        <Box ref={containerRef}>
+          <ListBoxRowColumn index={0} style={style} data={data} />
+          <ListBoxRowColumn index={1} style={style} data={data} autoFocus />
+          <ListBoxRowColumn index={2} style={style} data={data} />
+        </Box>
+      </ThemeProvider>
+    );
+
+    const firstRowColumn = getRow(0);
+    const middleRowColumn = getRow(1);
+    const lastRowColumn = getRow(2);
+
+    fireEvent.keyDown(firstRowColumn, { keyCode: 39 });
```

Comment:
Can we use enums from https://github.com/qlik-oss/nebula.js/blob/8eabc258297da778fa1ed7b2708fc8e2a714f9fd/apis/nucleus/src/keys.js#L1 To make it easier to read.